modules/manage/partials/tiered-storage.adoc

```diff
@@ -575,9 +576,24 @@ Microsoft ABS/ADLS::
 +
 --
 
-
 include::manage:partial$azure-blob-limitations.adoc[]
 
+To configure access to an Azure container with a managed identity:
+
+. Configure an xref:manage:security/iam-roles.adoc#configuring-iam-roles[Azure managed identity].
+. Run the `rpk cluster config edit` command, then edit the following required properties:
++
+[,properties]
+----
+cloud_storage_enabled: true
+cloud_storage_credentials_source: azure_vm_instance_metadata
+cloud_storage_azure_managed_identity_id: <managed-identity-object-id>
```

Comment:
@andijcr @c4milo can we confirm that the value required for this config can be found by the user as described in the Azure docs here? https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp#list-user-assigned-managed-identities